src/ssb.js

```diff
@@ -198,11 +134,25 @@ module.exports = ({ offline }) => {
     );
   }
 
-  const customConfig = {
-    conn: {
-      autostart: !offline,
-    },
-  };
+  // Make a copy of `ssbConfig` to avoid mutating.
+  const customConfig = JSON.parse(JSON.stringify(ssbConfig));
+
+  // This is unnecessary when https://github.com/ssbc/ssb-config/pull/72 is merged
+  customConfig.connections.incoming.unix = [
+    { scope: "device", transform: "noauth" },
+  ];
+
+  // Only change the config if `--offline` is true.
+  if (offline === true) {
```

Comment:
When I run `npm run dev -- --offline` I get the following error:

```
/Users/nickwynja/Projects/oasis/src/index.js:107
    throw err;
    ^

TypeError: Cannot set property 'autostart' of undefined
    at module.exports (/Users/nickwynja/Projects/oasis/src/ssb.js:147:33)
    at Object.<anonymous> (/Users/nickwynja/Projects/oasis/src/index.js:139:16)
```

Reply:
On it!